pxtlib/jsoutput.ts

```diff
@@ -273,7 +273,9 @@ namespace pxt.blocks {
     export interface SourceInterval {
         id: string;
         start: number;
+        startPos: number;
```

Comment:
please document the difference with start.

Reply:
ty, added clarification